Makefile

```diff
@@ -97,7 +97,7 @@ test-contracts-as: build-contracts-rs build-contracts-as
 	$(DISABLE_LOGGING) $(CARGO) test $(CARGO_FLAGS) --manifest-path "execution_engine_testing/tests/Cargo.toml" --features "use-as-wasm" -- --ignored
 
 .PHONY: test-contracts
-test-contracts: test-contracts-rs test-contracts-as
+test-contracts: test-contracts-rs
```

Comment:
Can you stamp a #TODO in these three places to serve as a reminder to bring the checks back some day?